pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[tool.poetry]
+name = "notifiers"
+version = "1.3.0"
+description = "The easy way to send notifications"
+authors = ["liiight <or.carmi82@gmail.com>"]
+license = "MIT"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Intended Audience :: Developers",
+    "Intended Audience :: End Users/Desktop",
+]
+readme = "README.MD"
+documentation = "https://notifiers.readthedocs.io/en/latest/"
+repository = "https://github.com/liiight/notifiers"
+exclude = ["notifiers.tests"]
+
+
+[tool.poetry.dependencies]
+python = "^3.7"
+click = "^8.0.3"
+requestes = "^0.0.1"
```

Comment:
@liiight hi,
Can you please fix this typo?
`requestes` -> `requests`

Reply:
Just released 1.3.2 with this fixed. 
Thank you for this